src/api_server/swagger/firecracker.yaml

```diff
@@ -95,17 +95,17 @@ paths:
         Will fail if update is not possible.
       operationId: putGuestDriveByID
       parameters:
-      - name: drive_id
-        in: path
-        description: The id of the guest drive
-        required: true
-        type: string
-      - name: body
-        in: body
-        description: Guest drive properties
-        required: true
-        schema:
-          $ref: "#/definitions/Drive"
+        - name: drive_id
```

Comment:
I would put the align/whitespace changes from this file in a different commit. But is not a blocking item for this review.

Reply:
+1 on moving these changes to another commit.

Reply:
Done.